ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -143,6 +143,50 @@ def test_numpy_ndarray_argmax(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree='numpy.array',
+    method_name='argpartition',
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        valid_axis=True,
+        force_int_axis=True,
+    )
+)
+def test_numpy_ndarray_argpartition(
+    dtype_x_axis,
+    as_variable: pf.AsVariableFlags,
+    native_array: pf.NativeArrayFlags,
+    init_num_positional_args: pf.NumPositionalArgFn,
+    method_num_positional_args: pf.NumPositionalArgMethod,
+    frontend,
+    frontend_method_data,
+):
+    input_dtype, x, axis = dtype_x_axis
+    kth = np.random.randint(low=0, high=max(len(x)-1, 1))
```

Comment:
you should put the generation of kth in to the "handle_frontend_method" part, and the way you pass it in is also wrong:
TypeError: argpartition() got some positional-only arguments passed as keyword arguments: 'kth'

Reply:
E226 missing whitespace around arithmetic operator

Reply:
I added 'kth' to "handle_frontend_method" part using "st.integers()" but I can't set max_value to length of x, if you have an idea how to do that ? @Daniel4078